content/en/versioncontrol.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Definition Template
+status: Feedback Appreciated
+category: Technology
+---
+
+## What it is
+Source control (or version control) is the practice of tracking and managing changes to code. To explain further, it is a system that records changes to a file or set of files over time so that you can recall specific versions later. 
+
+## Problem it addresses
+If you are a graphic or web designer (literally everyone who writes code) and want to keep every version of an image or layout, a Version Control System (VCS) is a very wise thing to use. It allows you to revert selected files back to a previous state, revert the entire project back to a previous state, compare changes over time, see who last modified something that might be causing a problem, who introduced an issue and when, and more.
```

Comment:
This is not the problem, you are already explaining how it helps which goes in the next section :)  The problem it addresses should talk about what it was like before there was version control. Ideally, you don't even mention the word "version control" in this section. 

Also, graphic or web designers generally don't code. Are you referring to front-end developers? There are VCS for designers but I think we should focus on developers in this glossary.

Reply:
Sure, I'll change it :)